src/bun.js/webcore/blob.zig

```diff
@@ -109,6 +109,10 @@ pub const Blob = struct {
     globalThis: *JSGlobalObject = undefined,
 
     last_modified: f64 = 0.0,
+    /// Blob name will lazy initialize when getName is called, but
+    /// we must be able to set the name, and we need to keep the value alive
+    /// https://github.com/oven-sh/bun/issues/10178
+    name: ?bun.String = null,
```

Comment:
this doesn't need to be optional, check if .tag == .Dead

Reply:
fixed: https://github.com/oven-sh/bun/commit/7689a6289d8025c234c571e59b50245b03b363b6